TASK/TaskVehicleDriveToCoordLongrange.md

```diff
@@ -10,6 +10,34 @@ void TASK_VEHICLE_DRIVE_TO_COORD_LONGRANGE(Ped ped, Vehicle vehicle, float x, fl
 
 You can let your character drive to the destination at the speed and driving style you set. You can use map marks to set the destination.
 
+### Flags
+enum EDriveBehaviorFlags {
+  DF_StopForCars = 1,
+  DF_StopForPeds = 2,
+  DF_SwerveAroundAllCars = 4,
+  DF_SteerAroundStationaryCars = 8,
+  DF_SteerAroundPeds = 16,
+  DF_SteerAroundObjects = 32,
+  DF_DontSteerAroundPlayerPed = 64,
+  DF_StopAtLights = 128,
+  DF_GoOffRoadWhenAvoiding = 256,
+  DF_DriveIntoOncomingTraffic = 512,
+  DF_DriveInReverse = 1024,
+  DF_UseWanderFallbackInsteadOfStraightLine = 2048,
```

Comment:
You could add `If pathfinding fails, navigate randomly instead of moving along a straight line.`